.github/workflows/remark.yml

```diff
@@ -22,7 +22,7 @@ jobs:
 
           if [[ ! -z ${changed_files_remark} ]]; then
             npm ci
-            npm install remark-cli vfile-reporter-json
+            npm install remark-cli@9.0.0 vfile-reporter-json@2.0.2
```

Comment:
What is a reason to have the specific version?

Reply:
As practice has shown when updating the ```vfile-reporter-json``` module to version 3.0.0 ```remark-cli``` couldn't create json report (PR #3156 ). Fixing the version for ```remark-cli``` I think it will not be superfluous.